app/controllers/catalog_controller.rb

```diff
@@ -1435,8 +1435,8 @@ def default_entry_point(prov_type)
     edit_new = @edit[:new]
     klass = class_service_template(prov_type)
     edit_new[:fqname] = klass.default_provisioning_entry_point
-    edit_new[:retire_fqname] = klass.default_retirement_entry_point if klass.respond_to?(:default_retirement_entry_point)
-    edit_new[:reconfigure_fqname] = klass.default_reconfiguration_entry_point if klass.respond_to?(:default_reconfiguration_entry_point)
+    edit_new[:retire_fqname] = klass.default_retirement_entry_point
+    edit_new[:reconfigure_fqname] = klass.default_reconfiguration_entry_point
```

Comment:
If we are not passing `service_type` from the caller (since it is not needed at this time) the parameter should be removed from the method definitions.